tests/conftest.py

```diff
@@ -40,7 +43,10 @@ def comparison_environments(tmp_path_factory, golden_dirs):
             comparison_dirs.append(path)
 
     comparison_apis = [hyp3_sdk.HYP3_PROD, hyp3_sdk.HYP3_TEST]
-    return list(zip(comparison_dirs, comparison_apis))
+    yield list(zip(comparison_dirs, comparison_apis))
```

Comment:
```suggestion

    yield list(zip(comparison_dirs, comparison_apis))

```
I think I'd prefer a bit of space around the yeild as it kinda a big deal